### ISSUE TITLE
Changing the bar menu icon by the colored pictogram used for status

### ISSUE THREAD

**Issue by ghost** [ygini/Hello-IT#10]
It will be great to be able to replace the bar menu icon by the same pictogram used to show the status of a service.
- If one service is orange or red, the icon becomes orange.
- If all services are down the icon becomes red.
- If everything is working the icon stays green.

**Comment by ygini** [comment]
@WardsParadox are you OK to decline your MenuBar logo in different colors? I've tried to simple use the fill feature in GIMP but the result is really bad due to different tint of black.

The main idea is to have a red, orange and gray version of the logo to display the global services state

**Comment by WardsParadox** [comment]
Not sure what you mean by decline, but I can easily wip something up for you. I used Inkscape with them so that they can be scaled up or down easily.
Any ideas for specific colors (ie: a specific red tone)?

**Comment by ygini** [comment]
I'm sorry, I've made a wrong translation, I mean develop of range of version.

Regarding the color tone, using the one used for service status could be great.

Look at Unavailable, Error and Warning image here https://github.com/ygini/Hello-IT/tree/master/HITDevKit/HITDevKit

**Comment by WardsParadox** [comment]
Alright. I'll work on them during lunch and submit them via Pull Request as well as in this issue.

**Comment by WardsParadox** [comment]
This would be a good time to add dark mode detection. I think I might have pieced together that plus the icon. Do you want the "it" still in the text @ygini ? With the smaller icon and color fills, it looks a little out of place in my opinion.

**Comment by WardsParadox** [comment]
![overviewofhitstatusbars](https://cloud.githubusercontent.com/assets/8117192/13685148/4bca82c2-e6cc-11e5-8765-ce2a290c47c3.png)

Here is an overview. I adjusted the black a little bit to more closely match the os x menu bar color from Yosemite + El Cap. I also added dark mode icons to the mix and I am working on some code for it.

**Comment by ygini** [comment]
I really like the dark mode idea. Thanks a lot for that. 

Regarding the IT or not inside, I don't know. We can have both depending if the icon can reflect a state or not. 

Regular icon with IT inside when we don't have any substate and colorized version when we have one. 

Don't forget that we need regular and @2x version for all version.

**Comment by WardsParadox** [comment]
Alright. I named the green dot statusbar-alt for now. All sized (including the @2x variants) are in the zip. Let me know if there are any issues :)
[Statusbar-icons2.zip](https://github.com/ygini/Hello-IT/files/168070/Statusbar-icons2.zip)

**Comment by WardsParadox** [comment]
Last code I saw that didn't use image templates was this:

```
    NSString *osxMode = [[NSUserDefaults standardUserDefaults] stringForKey:@"AppleInterfaceStyle"];
```

if normal mode the value is `nil`, if dark mode is enabled it is `Dark`
I don't know swift or object c so I am a little lost on that
Top answer:
http://stackoverflow.com/questions/25207077/how-to-detect-if-os-x-is-in-dark-mode

**Comment by ygini** [comment]
Solved in v1.1 available now in the download page.

Thanks again @WardsParadox for the icon.
